dashboard/app/views/levels/_dialog.html.haml

```diff
@@ -48,14 +48,20 @@
         - elsif app == 'text_match' && data['answers'].blank?
           =t('text_match.open_response_title')
         - else
-          =t("#{app == 'match' ? 'match' : 'multi'}.correct_title")
+          - if data['longassessment']
+            =t("#{app == 'match' ? 'match-longassessment' : 'multi-longassessment'}.correct_title")
+          - else
+            =t("#{app == 'match' ? 'match' : 'multi'}.correct_title")
```

Comment:
can this be replaced with 
````
=t("#{app}.correct_title")
````
or do apps that are neither multi nor match use the multi title?